dictionaries/CallMap.php

```diff
@@ -9625,7 +9625,7 @@
 'rand\'1' => ['int'],
 'random_bytes' => ['non-empty-string', 'length'=>'positive-int'],
 'random_int' => ['int', 'min'=>'int', 'max'=>'int'],
-'range' => ['array', 'start'=>'mixed', 'end'=>'mixed', 'step='=>'int|float'],
+'range' => ['non-empty-array', 'start'=>'string|int|float', 'end'=>'string|int|float', 'step='=>'int<1, max>|float<1, max>'],
```

Comment:
`float` has no range support, so it should be just 
```suggestion
'range' => ['non-empty-array', 'start'=>'string|int|float', 'end'=>'string|int|float', 'step='=>'int<1, max>|float'],
```